### PR TITLE
timing out requests

### PR DESCRIPTION
**Pull request by wheresrhys**
In line with what Niklas told us about 5s limit on request handling

@phamann

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -188,6 +188,19 @@ module.exports = function(options) {
...
 
+	app.use((req, res) => {
+		setTimeout(() => {
+			nLogger.error('RESPONSE_TIMED_OUT', {
+				url: req.url
+			});
+			try {
+				res.sendStatus(504)
+			} catch (e) {
+				// avoids sending a 'headers already sent' error to sentry
```
Comment (matthew-andrews) [inline main.js]: hmm, i think this isn't where that error would be sent from

Comment (matthew-andrews) [inline main.js]: isn't just*

Comment (matthew-andrews) [inline main.js]: also can this be an optional feature for now while we test it in a non-platinum app?

Comment (wheresrhys) [inline main.js]: sure

**Pull request closed without merging**